homeassistant/helpers/template.py

```diff
@@ -884,6 +884,16 @@ def ordinal(value):
     )
 
 
+def from_json(value):
+    """Convert a string to a JSON object."""
+    return json.loads(value)
+
+
+def to_json(value):
+    """Convert an object to a JSON string."""
+    return json.dumps(value)
```

Comment:
What is the use case for `to_json` ? Template outputs are always strings inside Home Assistant, they are not allowed to be JSON and then be parsed.

Reply:
My original description says it all. When I tried to pass the"trigger.json" object from a webhook automation to a script as a parameter, it came through as a string that is not valid JSON. And there was no way to parse it back to an object.

I can call to_json when passing the parameter and then from_json in the script to convert it back to a usable object. Otherwise this is currently not possible as far as I can tell